packages/experimental/Button/src/Button.types.ts

```diff
@@ -106,6 +113,18 @@ export interface ButtonProps extends ButtonCoreProps {
 
   /** Sets style of button to a preset size style  */
   size?: ButtonSize;
+
+  /**
+   * Button shape: 'rounded' | 'circular' | 'square'
+   * @defaultvalue rounded
+   */
+  shape?: ButtonShape;
+
+  /**
+   * Icon can be placed before or after Button's content.
+   * @default 'before'
+   */
+  iconPosition?: 'before' | 'after';
```

Comment:
I thought the options for this prop were `left` and `right`, did we resolve this difference with web yet? Or did we decide to use `before` and `after` instead?

Reply:
I took it from Web. I remember that we didn't like the name left/right. I see that I didn't update the type in the doc, it's still 'left' | 'right' @ejlayne, is 'before' | 'after' good name for iconPosition?

Reply:
'before' | 'after' is correct. I updated the doc